webxrlayers-1.bs

```diff
@@ -263,8 +264,24 @@ enable optical chromatic aberration correction for the layer. Only valid if this
 
 ISSUE: describe what chromaticAberrationCorrection does.
 
+The <dfn attribute for="XRCompositionLayer">needsRedraw</dfn> attribute signals that the {{XRCompositionLayer}} SHOULD be
```

Comment:
Really minor nit: We should only use all-caps SHOULD/MUST/MAY etc when describing UA behavior, since that's the only thing this spec can actually control. Since this describes page behavior it needs to be simply "should"